src/client/client.ts

```diff
@@ -10,32 +10,70 @@ interface ClientConfig {
   chain?: {
     id: number;
     name: string;
-    rpcUrls: {default: {http: readonly string[]}};
-    nativeCurrency: {name: string; symbol: string; decimals: number};
-    blockExplorers?: {default: {name: string; url: string}};
+    rpcUrls: { default: { http: readonly string[] } };
+    nativeCurrency: { name: string; symbol: string; decimals: number };
+    blockExplorers?: { default: { name: string; url: string } };
   };
-  endpoint?: string; // Optional: Custom RPC endpoint override
-  account?: Account;
+  endpoint?: string; // Custom RPC endpoint
+  accountOrAddress?: Account | Address;
```

Comment:
@epsjunior  this is a breaking change that will require updates in the boilerplate, studio, Intelligent oracle and every project that uses genlayer-js. I would avoid it.